### ISSUE TITLE
Chrome build of Configurator doesn't work

### ISSUE THREAD

**Issue by digitalentity** [iNavFlight/inav-configurator#790]
Builds correctly, however doesn't start properly:

![image](https://user-images.githubusercontent.com/11059099/58906900-498fdb80-870d-11e9-9291-787e389d54c6.png)

Console:

![image](https://user-images.githubusercontent.com/11059099/58906923-544a7080-870d-11e9-8745-6ab1eefa5f8d.png)

```
script.js:2391 Uncaught ReferenceError: global is not defined
    at script.js:2391
```

**Comment by DzikuVx** [comment]
it's the nw-dialog.js

`_context: typeof global.DOMDocument === 'undefined' ? document : global.DOMDocument,`

**Comment by DzikuVx** [comment]
fixed in #791
